modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb

```diff
@@ -98,16 +101,60 @@
       Timecop.freeze(Time.zone.parse('2020-01-01T08:00:00Z'))
       path = described_class.new.generate_pdf(extra_higher_level_review.id)
       expected_path = Rails.root.join('modules', 'appeals_api', 'spec', 'fixtures', 'expected_200996_extra.pdf')
-      generated_pdf_md5 = Digest::MD5.digest(File.read(path))
-      expected_pdf_md5 = Digest::MD5.digest(File.read(expected_path))
+
+      generated_md5s = pdf_to_md5s(path, extra_higher_level_review.id)
+      expected_md5s = pdf_to_md5s(expected_path, extra_higher_level_review.id)
+
+      expect(generated_md5s).to eq(expected_md5s)
+
       File.delete(path) if File.exist?(path)
-      expect(generated_pdf_md5).to eq(expected_pdf_md5)
       Timecop.return
     end
   end
 
   private
 
+  # Converts PDF at path to array of md5's, one md5 for each page of the document.
+  # Problem: Taking an md5 of the PDF itself causes a mis-match between the generated and expected PDFs due to
+  # differences in metadata.
+  #
+  # @param [String] path The path to the PDF
+  # @param [String] identifier The unique identifier of the higher level review
+  # @return [[String]] Array of md5's
+  def pdf_to_md5s(path, identifier)
```

Comment:
@bastosmichael do we have a good place to put spec helper methods that can be shared across modules or should we have the duplicate code in this case of these two methods: `pdf_to_md5s` and `pdf_to_image`?

Reply:
sounds good.